go.mod

```diff
@@ -7,44 +7,44 @@ toolchain go1.22.3
-	k8s.io/apiserver => k8s.io/apiserver v0.30.1
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.30.1
-	k8s.io/client-go => k8s.io/client-go v0.30.1
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.30.1
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.30.1
-	k8s.io/code-generator => k8s.io/code-generator v0.30.1
-	k8s.io/component-base => k8s.io/component-base v0.30.1
-	k8s.io/cri-api => k8s.io/cri-api v0.30.1
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.30.1
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.30.1
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.30.1
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.30.1
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.30.1
-	k8s.io/kubectl => k8s.io/kubectl v0.30.1
-	k8s.io/kubelet => k8s.io/kubelet v0.30.1
-	k8s.io/kubernetes => k8s.io/kubernetes v1.30.1
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.30.1
-	k8s.io/metrics => k8s.io/metrics v0.30.1
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.30.1
+	k8s.io/api => k8s.io/api v0.31.1
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.31.1
+	k8s.io/apimachinery => k8s.io/apimachinery v0.31.1
+	k8s.io/apiserver => k8s.io/apiserver v0.31.1
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.31.1
+	k8s.io/client-go => k8s.io/client-go v0.31.1
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.31.1
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.31.1
+	k8s.io/code-generator => k8s.io/code-generator v0.31.1
+	k8s.io/component-base => k8s.io/component-base v0.31.1
+	k8s.io/cri-api => k8s.io/cri-api v0.31.1
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.31.1
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.31.1
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.31.1
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.31.1
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.31.1
+	k8s.io/kubectl => k8s.io/kubectl v0.31.1
+	k8s.io/kubelet => k8s.io/kubelet v0.31.1
+	k8s.io/kubernetes => k8s.io/kubernetes v1.31.1
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.31.1
+	k8s.io/metrics => k8s.io/metrics v0.31.1
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.31.1
 )
 
 require (
 	github.com/aws/aws-sdk-go-v2/config v1.27.18
 	github.com/aws/aws-sdk-go-v2/service/licensemanager v1.25.9
 	github.com/aws/aws-sdk-go-v2/service/sts v1.28.12
 	github.com/google/uuid v1.6.0
-	github.com/prometheus/client_model v0.5.0
-	github.com/prometheus/common v0.45.0
-	github.com/rancher/lasso v0.0.0-20240603075835-701e919d08b7
-	github.com/rancher/rancher/pkg/apis v0.0.0-20240613110601-c21147a009c6
-	github.com/rancher/wrangler/v3 v3.0.0-rc2
+	github.com/prometheus/client_model v0.6.1
+	github.com/prometheus/common v0.55.0
+	github.com/rancher/lasso v0.0.0-20240924233157-8f384efc8813
+	github.com/rancher/rancher/pkg/apis v0.0.0-20241001074650-6b63832fda0f
+	github.com/rancher/wrangler/v3 v3.0.0
```

Comment:
Is there a specific reason to use v3.0.0 version of wrangler? I see this commit 
https://github.com/rancher/wrangler/commit/3e8181532ae254bf863dd8246ab711910d84f1c5 
which is not available in v3.0.0.

Reply:
I wasn't sure which version of wrangler to use, so I went with the latest tag available. Should we be using v3.0.1-rc2 instead?

Reply:
For now we could use v3.0.1-rc2 but I think it's best to the v3.0.1 once available.